src/main/java/org/sagebionetworks/web/client/widget/entity/browse/FilesBrowser.java

```diff
@@ -1,53 +1,19 @@
 package org.sagebionetworks.web.client.widget.entity.browse;
 
-import com.google.gwt.user.client.rpc.AsyncCallback;
 import com.google.gwt.user.client.ui.IsWidget;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
-import java.util.ArrayList;
-import java.util.List;
-import org.sagebionetworks.repo.model.EntityChildrenRequest;
-import org.sagebionetworks.repo.model.EntityChildrenResponse;
-import org.sagebionetworks.repo.model.EntityType;
-import org.sagebionetworks.web.client.GlobalApplicationState;
-import org.sagebionetworks.web.client.SynapseJavascriptClient;
-import org.sagebionetworks.web.client.cookie.CookieProvider;
-import org.sagebionetworks.web.client.security.AuthenticationController;
 import org.sagebionetworks.web.client.utils.CallbackP;
 import org.sagebionetworks.web.client.widget.SynapseWidgetPresenter;
-import org.sagebionetworks.web.client.widget.clienthelp.ContainerClientsHelp;
-import org.sagebionetworks.web.client.widget.entity.file.AddToDownloadListV2;
 
-public class FilesBrowser
-  implements SynapseWidgetPresenter, FilesBrowserView.Presenter {
+public class FilesBrowser implements SynapseWidgetPresenter {
```

Comment:
Removed logic related to rendering a download menu, now handled by EntityActionController.